main/src/cgeo/geocaching/ui/dialog/CoordinatesInputDialog.java

```diff
@@ -546,6 +561,33 @@ public void onClick(final View v) {
         }
     }
 
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        dismiss();
+    }
+
+    private class CalculateListener implements View.OnClickListener {
+
+        @Override
+        public void onClick(final View v) {
+            if (cacheCoords == null) {
+                final AbstractActivity activity = (AbstractActivity) getActivity();
+                activity.showToast(activity.getString(R.string.err_location_unknown));
+                return;
+            }
+
+
+            CalcState theState = ((CalculateState) getActivity()).fetchCalculatoState();
+            final CoordinatesCalculateDialog calculateDialog = CoordinatesCalculateDialog.getInstance(gp, theState);
+
+             // Assign this fragment as the target fragment so the calculate dialog can automatically close this one on completion
```

Comment:
Oh, I was not talking about the blank line, just the single additional space for indentation.